src/plugins/dashboard/public/application/legacy_app.js

```diff
@@ -165,7 +165,7 @@ export function initDashboardApp(app, deps) {
           };
 
           $scope.editItem = ({ editUrl }) => {
-            history.push(deps.addBasePath(editUrl));
+            history.push(editUrl);
           };
           $scope.viewItem = ({ viewUrl }) => {
             history.push(deps.addBasePath(viewUrl));
```

Comment:
Does view url still needs adding base path here if `yarn start` has already added 3 digit base path as mentioned here https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3861 ?

Reply:
View url still functionals as a href so it's fine.